pkg/addons/velero/integration/hostcabundle_test.go

```diff
@@ -0,0 +1,90 @@
+package integration
+
+import (
+	"context"
+	"strings"
+	"testing"
+
+	"github.com/replicatedhq/embedded-cluster/pkg/addons/velero"
+	"github.com/replicatedhq/embedded-cluster/pkg/helm"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/utils/ptr"
+	"sigs.k8s.io/yaml"
+)
+
+func TestHostCABundle(t *testing.T) {
```

Comment:
nit: why is this an integration test if it's mocking everything and only testing the velero component?

Reply:
it really pulls the chart and uses the helm client to render the chart